dockers/broad/verify_bam_id/docker_build.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+set -e
+
+# VerifyBamID is based off a specific githash and unlikely to change
+VERIFY_BAM_ID_VERSION="c1cba76e979904eb69c31520a0d7f5be63c72253"
+TIMESTAMP=$(date +"%s")
+DIR=$(cd $(dirname $0) && pwd)
+
+# Registries and tags
+GCR_URL="us.gcr.io/broad-gotc-prod/verify-bam-id"
+DOCKERHUB_URL=""
```

Comment:
probably not needed after all

Reply:
was just gonna leave them in there until we figure out what we're doing but probably does make more sense to remove them for now